### PR TITLE
Added open file for Hamburger Menu + / for gitignore node_modules

### PR DESCRIPTION
**Pull request by dashasandler**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 allure-results
 allure-report
-node_modules
+/node_modules
```
Comment (Burdishka) [inline .gitignore:3]: Why do we need this "/"?
```suggestion
node_modules
```

File: test/pageobjects/HamburgerMenu.page.js [hunk 2]
```diff
@@ -25,6 +25,8 @@ class HamburgerMenuPage extends Page{
     get logOut(){
         return $("#logout");
     }
-
```
Comment (Burdishka) [inline test/pageobjects/HamburgerMenu.page.js:28]: Just an idea - we need to write our code in one style. As for me, one empty line is fine to separate blocks of code.

**Review by Burdishka — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (dashasandler) [reply .gitignore:3]: @ErikHP так у себя в проектах видел

File: test/pageobjects/HamburgerMenu.page.js (hunk 2 above)
Comment (dashasandler) [reply test/pageobjects/HamburgerMenu.page.js:28]: Ok! Will do. But need help how properly change this. Do I have to close this PR and after changes do PR again,?

File: .gitignore (hunk 1 above)
Comment (ErikHP) [reply .gitignore:3]: @Burdishka that how we did in our projects) Check my page

File: test/pageobjects/HamburgerMenu.page.js (hunk 2 above)
Comment (ErikHP) [reply test/pageobjects/HamburgerMenu.page.js:28]: There is a plugin called Beautify. We can use that to make it look beautiful and identical in style. P.S. we used it in our projects as well

Comment (Burdishka) [reply test/pageobjects/HamburgerMenu.page.js:28]: @ErikHP so if you could use it, then use it. I don't see that you used it in your [#15](https://github.com/QA4Everyone3/group_bug_crushers/pull/15) :)

**Pull request closed without merging**